### DISCUSSION TITLE
Can you add PHP to the languages list?

### DISCUSSION THREAD

**Discussion by amiria703** [monkeytypegame/monkeytype#2836] [Ideas]
Hi,
Is it possible to include PHP in the languages list?

**Comment by rizwanmustafa** [comment]
Yes, you can add PHP to the language: `code`. 
Many of the famous programming languages have already been added eg `code python` .`code javascript`.
If you need help regarding contributing to the project, I recommend you checkout the `CONTRIBUTING.md`
